[Compilation] Current master not compiling on Windows

See title. The following error occures:

       "C:\software_development\kratos\cmake_build\INSTALL.vcxproj" (Standardziel) (1) ->
       "C:\software_development\kratos\cmake_build\ALL_BUILD.vcxproj" (Standardziel) (3) ->
       "C:\software_development\kratos\cmake_build\kratos\KratosCore.vcxproj" (Standardziel) (14) ->
       (ClCompile Ziel) ->
         C:\software_development\kratos\kratos\utilities\os_utilities.cpp(19): fatal error C1083: Cannot open include file: 'unistd.h': No such file or directory [C:\software_development\kratos\cmake_build\kratos\KratosCore.vcxproj]

    16 Warnung(en)
    1 Fehler

It seems due to the changes made in #4911. Despite it is a Windows compilation, it tries to include this file. Some things were already discussed here: #4910. From a quick search, it seems for me a problem related to the #ifdef _WIN32 statement. 

@loumalouomega Did you try the compilation on windows? Or can you reproduce this error?

--- a/kratos/utilities/os_utilities.cpp
+++ b/kratos/utilities/os_utilities.cpp
@@ -12,7 +12,7 @@
 
 // System includes
 #include <stdio.h>  /* defines FILENAME_MAX */
-#ifdef KRATOS_COMPILED_IN_WINDOWS
+#ifdef _WIN32
 #include <direct.h>
 #define GetCurrentDir _getcwd
 #else

